lzy/portal/src/main/java/ai/lzy/portal/slots/StdoutSlot.java

```diff
@@ -27,7 +29,9 @@ public class StdoutSlot extends LzySlotBase implements LzyOutputSlot {
 
     private final Map<String, StdoutInputSlot> task2slot = new HashMap<>();
     private final Map<String, String> slot2task = new HashMap<>();
-    private final CircularFifoQueue<String> buffer = new CircularFifoQueue<>(1024);
+
+    // TODO(artolord) replace this buffer with persistent queue or file to prevent OOM
+    private final LinkedBlockingQueue<String> buffer = new LinkedBlockingQueue<>(1024);
```

Comment:
hmm... I believe that the loss of some messages (in case of CircularQueue) is much better then OOM  :)